### DISCUSSION TITLE
Hide Basestation LED and KeyPad LED lights from everyone

### DISCUSSION THREAD

**Discussion by bmgreenwood** [dgreif/ring#1208] [Q&A]
I have been struggling to find a way to hide the lights at all from HomeBridge for the ring alarm basestation and keypads. It always shows, and I cant ask Siri to turn on/off the lights in a particular room without those being involved. How do I hide them?

**Comment by smockle** [comment]
You can use [`hideDeviceIds`](https://github.com/dgreif/ring/tree/main/packages/homebridge-ring#optional-parameters) to hide those lights. The device identifiers are logged when you start homebridge-ring.
